test/fs.js

```diff
@@ -34,3 +35,25 @@ testMkdirp.test('create 2 directories without mode', test => {
     test.end();
   });
 });
+
+metatests.test('rmdirp test', test =>
+  fs.mkdir(RMDIRP_TEST_DIR, err => {
+    if (err && err.code !== 'EEXISTS') {
+      test.bailout(err);
+    }
+    fs.mkdir(path.join(RMDIRP_TEST_DIR, 'subdir1'), err => {
+      if (err && err.code !== 'EEXISTS') {
+        test.bailout(err);
+      }
+      rmdirp(path.join(RMDIRP_TEST_DIR, 'subdir1'), err => {
+        if (err && err.code !== 'EEXISTS') {
```

Comment:
I think `EEXISTS` should not be ignored here:  
```suggestion
        if (err) {
```